docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -63,8 +63,19 @@ Now that you have the tooling installed, we can start building our application.
 
 First, create a new project using:
 
+<!--
+
+Configure `git` email and name, otherwise `zkapp-cli` project generation will return a non-zero exit code.
+
+```sh
+$ git config --global user.email "test@example.com"
+$ git config --global user.name "Test"
+```
+
+-->
```

Comment:
Using code blocks in HTML comments, it's trivial to interleave the code blocks that the reader sees with code blocks that get executed by the runner (and thus tested), but which are hidden from the rendered doc.